Dockerfile

```diff
@@ -18,13 +18,12 @@ COPY . /go/src/github.com/square/ghostunnel
 
 # Build
 RUN cd /go/src/github.com/square/ghostunnel && \
-    GO111MODULE=on make clean ghostunnel && \
+    CGO_ENABLED=0 GOOS=linux GO111MODULE=on make clean ghostunnel && \
```

Comment:
This would break users of PKCS#11/HSMs, which is something I'd like to keep working in Docker.

Reply:
Why is it breaking it?

Reply:
Because PKCS#11 support requires CGO to be enabled.

Reply:
Ah, let me check... :)